pkg/project/project.go

```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA      = "n/a"
 	name        = "cluster-operator"
 	source      = "https://github.com/giantswarm/cluster-operator"
-	version     = "2.1.11-dev"
+	version     = "2.1.12-dev"
```

Comment:
For future use. Let's keep the semver version and rather modify the suffix. I would do `2.1.11-xh3b4sd`. That way everyone can still be sure it is my test version of `2.1.11`.